www/src/components/ReactPlayground.js

```diff
@@ -11,6 +11,7 @@ import React, {
   useState,
 } from 'react';
 import * as ReactBootstrap from 'react-bootstrap';
+import AccordionContext from 'react-bootstrap/AccordionContext';
```

Comment:
**question:** is this not available as `ReactBootstrap.AccordionContext` because it's not exported from the package index? if so, then we should just export it from the package index, instead of doing something special in the examples here

Reply:
That is correct. I didn't know if there was a reason that it wasn't exported as part of the index.

Reply:
yeah, so just add the export then

Reply:
Sounds good to me. done.